pkg/config/config_test.go

```diff
@@ -940,3 +941,15 @@ func TestPeerTagsEnv(t *testing.T) {
 	testConfig = SetupConfFromYAML("")
 	require.Equal(t, []string{"aws.s3.bucket", "db.instance", "db.system"}, testConfig.GetStringSlice("apm_config.peer_tags"))
 }
+
+func TestLogDefaults(t *testing.T) {
+
+	c := NewConfig("test", "DD", strings.NewReplacer(".", "_"))
+	require.Equal(t, 0, c.GetInt("log_file_max_rolls"))
```

Comment:
Is this still passing?

Reply:
yes. It should be zero for NewConfig without initmethod